app/src/main/java/com/eis/phoneringer/MainActivity.java

```diff
@@ -1,14 +1,127 @@
 package com.eis.phoneringer;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.Manifest;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.os.Build;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
 
+import com.eis.phoneringer.structure.AppManager;
+import com.eis.phoneringer.structure.PasswordManager;
+import com.eis.phoneringer.structure.ReceivedMessageListener;
+import com.eis.phoneringer.structure.RingCommand;
+import com.eis.smslibrary.SMSHandler;
+import com.eis.smslibrary.SMSMessage;
+import com.eis.smslibrary.SMSPeer;
+import com.eis.smslibrary.exceptions.InvalidSMSMessageException;
+import com.eis.smslibrary.exceptions.InvalidTelephoneNumberException;
+import com.eis.smslibrary.listeners.SMSSentListener;
+
+/**
+ * Test app: through this MainActivity we can test out our basic library for sending ring commands
+ *
+ * @author Alberto Ursino
+ */
 public class MainActivity extends AppCompatActivity {
 
+    private static Button ringButton = null;
+    private static EditText phoneNumber = null;
+    private static final String DEFAULT_PASSWORD = "password";
+    private PasswordManager passwordManager = null;
```

Comment:
Why are you assigning `null` on a `null` variable? Looks useless to me

Reply:
For a better clarity of the code. 
Should I to remove them?